### DISCUSSION TITLE
SSD1315 versus SSD1306

### DISCUSSION THREAD

**Discussion by profharris** [olikraus/u8g2#2540] [General]
I've noticed there are no SSD1315 Constructor References.  Several of my newer small OLED devices purchases now come with the new SSD1315 driver IC instead of the SSD1306.  Has anyone researched the compatibility (backward) between OLEDs with the SSD1315 controller versus the SSD1306 (or other) controller?  -profharris

**Comment by olikraus** [comment]
I always assumed that there is a hardware difference, but hardware register set should be the same.

**Comment by Rahul-Karshak** [comment]
@olikraus - I’ve used this library(_and u8g_) with SSD1306 and SH1106 drivers.
While checking the Waveshare wiki ([**0.49inch OLED Module**](https://www.waveshare.com/wiki/0.49inch_OLED_Module)), I noticed that it actually uses the SSD1315 display driver, which isn’t included (_though SSD1316 is_). At first, I assumed SSD1316 would be compatible, but after a bit of research, I found that SSD1315 is actually closer to SSD1306 than to SSD1316.

So, @profharris - Did it work for you?
